src/main/java/newbank/server/Customer.java

```diff
@@ -40,15 +40,31 @@ public String getUsername() {
   public String getPassword() {
     return password;
   }
-  
+
+  public boolean hasAccount(final String accountName) {
+    return accounts
+        .stream()
+        .filter(a -> a.getName().equalsIgnoreCase(accountName))
+        .findFirst()
+        .isPresent();
```

Comment:
You can simplify this using `Stream#anyMatch`:
`return accounts.stream().anyMatch(a -> a.getName().equalsIgnoreCase(accountName));`

Reply:
anyMatch returns a boolean, not what I want here.

Kindest Regards,
Jules